src/web/run.rs

```diff
@@ -46,7 +46,9 @@ pub(crate) fn run_web(args: &RunArgs) -> anyhow::Result<()> {
         match webbrowser::open(&url) {
             Ok(()) => info!("Your app is running at <{url}>!"),
             Err(error) => {
-                error!("Failed to open the browser automatically, open the app at <{url}>. (Error: {error:?}");
+                error!(
+                    "Failed to open the browser automatically, open the app at <{url}>. (Error: {error:?}"
```

Comment:
```suggestion
                    "Failed to open the browser automatically, open the app at <{url}>. (Error: {error:?})"
```

Completely unrelated, but looks like this is missing a closing parentheses. Mind as well fix it here :)